tests/conftest.py

```diff
@@ -179,6 +179,16 @@ def application_docker_images(docker_client: DockerClient) -> Iterable[Mapping[s
     images = {}
     for runtime in os.listdir(str(CONTAINERS_DIRECTORY)):
         images[runtime], _ = docker_client.images.build(path=str(CONTAINERS_DIRECTORY / runtime), rm=True)
+
+        # for java - build image based on "j9"
+        if runtime == "java":
+            images[runtime + "_j9"], _ = docker_client.images.build(
+                path=str(CONTAINERS_DIRECTORY / runtime),
+                rm=True,
+                buildargs={"JAVA_BASE_IMAGE": "adoptopenjdk/openjdk11-openj9"},
```

Comment:
```suggestion
                buildargs={"JAVA_BASE_IMAGE": "adoptopenjdk/openjdk8-openj9"},
```
because we test HotSpot at JDK8.

Reply:
Done